homeassistant/components/sensor/forecast.py

```diff
@@ -134,11 +137,20 @@ def update(self):
         import forecastio
 
         self.forecast_client.update()
-        data = self.forecast_client.data
+        data = self.forecast_client.data.currently()
+        data_minutely = self.forecast_client.data.minutely()
```

Comment:
Or does it work just like `currently` and does it not cause an extra request to the server ?